conan/api/model/list.py

```diff
@@ -36,6 +36,10 @@ def merge(self, other):
         for k, v in other.lists.items():
             self.lists.setdefault(k, PackagesList()).merge(v)
 
+    def keep_outer(self, other):
```

Comment:
I'd prefer this processing to be local inside ``load_graph()``

Reply:
Yeah, but what about the `PackagesList` part? Also embedded in the load graph?